src/rsz/src/RepairSetup.cc

```diff
@@ -537,7 +539,19 @@ bool RepairSetup::repairPath(PathRef& path,
                  || (pair1.second == pair2.second && pair1.first > pair2.first);
         });
     // Attack gates with largest load delays first.
+    int repairs_per_iter = std::round(std::max(-path_slack * 10e9f, 1.0f));
```

Comment:
Is there an assumption that slack is in ns?  This is dependent on the Liberty units and may vary by PDK.

Reply:
It's just a constant that worked pretty well.
The final version was going to either have this factor configurable via a parameter, or have the number of repairs always be the maximum (10 here) for the first endpoint, and then linearly go down to 1 for the final endpoints.